src/auth/providers.js

```diff
@@ -56,8 +56,8 @@ function anonProvider(auth) {
 
 /** @namespace */
 function userPassProvider(auth) {
-  const providerRoute = auth.isAppClient() ? 'local/userpass' : 'providers/local-userpass';
-  const loginRoute = auth.isAppClient() ? 'local/userpass' : `${providerRoute}/login`;
+  const providerRoute = auth.isAppClient() ? 'providers/local-userpass' : 'providers/local-userpass';
+  const loginRoute = auth.isAppClient() ? `${providerRoute}/login` : `${providerRoute}/login`;
```

Comment:
is it worth adding a comment about the ternary weirdness? I could see this being a little confusing otherwise